src/test/java/de/funfried/netbeans/plugins/external/formatter/sql/jsqlformatter/JSQLFormatterServiceTest.java

```diff
@@ -41,11 +41,11 @@ public JSQLFormatterServiceTest(String name) {
 	public void testFormat() throws Exception {
 		final String text = "SELECT FOO FROM BAR WHERE FOO = 'BAR' ORDER BY FOO LIMIT 1\n";
 		final String expected = "SELECT foo\n" +
-				"FROM bar\n" +
-				"WHERE foo = 'BAR'\n" +
-				"ORDER BY foo\n" +
-				";\n" +
-				"";
+								"FROM bar\n" +
```

Comment:
The formatting seems to be wrong. Could you please try to use the Eclipse Formatter for Java with the template contained in the main folder of the sources?